client/my-sites/checkout/checkout/domain-details-form.jsx

```diff
@@ -380,78 +385,40 @@ export class DomainDetailsForm extends PureComponent {
 		);
 	}
 
-	renderAddressFields() {
-		return (
-			<div>
-				<Input
-					label={ this.props.translate( 'Address' ) }
-					maxLength={ 40 }
-					{ ...this.getFieldProps( 'address-1' ) }
-				/>
-
-				<HiddenInput
-					label={ this.props.translate( 'Address Line 2' ) }
-					text={ this.props.translate( '+ Add Address Line 2' ) }
-					maxLength={ 40 }
-					{ ...this.getFieldProps( 'address-2' ) }
-				/>
-			</div>
-		);
-	}
-
-	renderCityField() {
-		return <Input label={ this.props.translate( 'City' ) } { ...this.getFieldProps( 'city' ) } />;
-	}
-
-	renderStateField() {
-		const countryCode = formState.getFieldValue( this.state.form, 'countryCode' );
-
-		return (
-			<StateSelect
-				label={ this.props.translate( 'State' ) }
-				countryCode={ countryCode }
-				{ ...this.getFieldProps( 'state' ) }
-			/>
-		);
-	}
-
-	renderPostalCodeField() {
-		return (
-			<Input
-				label={ this.props.translate( 'Postal Code' ) }
-				{ ...this.getFieldProps( 'postal-code' ) }
-			/>
-		);
-	}
-
-	renderCountryDependentAddressFields( needsOnlyGoogleAppsDetails ) {
+	renderDomainContactDetailsFields() {
+		const { contactDetails } = this.props,
```

Comment:
We usually use a const every line rather than using run-on commas because it makes for cleaner diffs.

Reply:
I'll clean up the whole file in accordance with this. 👍 It was [inconsistent](https://github.com/Automattic/wp-calypso/blob/aa71c003fcea6011d67200f13e74869b3def4602/client/my-sites/checkout/checkout/domain-details-form.jsx#L472).